mbed_build/_internal/mbed_tools/export.py

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (C) 2020 Arm Mbed. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+"""Export top-level CMakeLists.txt command."""
+import pathlib
+
+import click
+
+from mbed_build.mbed_build import generate_cmakelists_file, write_cmakelists_file
+
+
+@click.command()
+@click.option(
+    "-o",
+    "--output_directory",
```

Comment:
Question, should we use `--output-directory` with a hyphen instead?

Reply:
edb1dde